lessons/06-html-layout/posts/post-1.html

```diff
@@ -0,0 +1,42 @@
+<html>
+
+<head>
+    <title>Supercalifragilisticexpialidocoius</title>
+</head>
+<header>
+
+    <body>
+        <h1><a href="../index.html">Greanncy's Blog</a></h1>
+    </body>
+</header>
+<main>
+    <article>
+        <h2>Supercalifragilisticexpialidocoius</h2>
+        <p><em><span title="Jan 06, 2019 03:34:41"> Jan 06, 2019</span></em></p>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusantium molestias sunt sed corrupti unde cum
+            repudiandae incidunt itaque quaerat? Doloribus voluptatum itaque sunt nisi dolor inventore perferendis esse
+            veniam fugit.</p>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+            <br />
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+            <hr />
+    </article>
+    <footer>
+        <p>- Greanncy Ramales</p>
+    </footer>
+</main>
```

Comment:
Oops! This looks good except we need to check where our `<body>` tag is at. Our body tag needs to wrap **all** content that is displayed on the website, so we need to place the `<header>` and `<main>` tags here inside of it